test/deprecated/random/RandomStreamInterface/choiceTestArray.good

```diff
@@ -0,0 +1,200 @@
+choiceTestArray.chpl:3: In function 'main':
```

Comment:
It seems to me that all of these RandomStreamInterface tests shouldn't be in test/deprecated/random but rather you should modify them to get them working since `choice` is merely unstable and not deprecated.

Reply:
Okay, I've moved this test back to the library folder, but kept the deprecation warnings in the good files for now.